src/flow-spec/Group.ts

```diff
@@ -0,0 +1,30 @@
+import {createFormattedDate, IContactGroup, IGroup, isGroup} from '..'
+
+export class Group implements IGroup {
+  constructor(public groupKey: string, public label?: string) {}
+
+  get __value__(): string {
+    return this.groupKey
+  }
+}
+
+export class ContactGroup extends Group implements IContactGroup {
+  updatedAt: string
+  deletedAt?: string | undefined
```

Comment:
Should just be `deletedAt?: string` as `deletedAt?: string`, `deletedAt?: string | undefined` and `deletedAt: string | undefined` all mean the same thing, but `deletedAt?: string` is the preferred TypeScript-esque way of doing it.